docs/user/pings/custom.md

```diff
@@ -58,6 +60,26 @@ override fun onCreate() {
 
 </div>
 
+<div data-lang="Swift" class="tab">
+
+In Swift, this object must be registered with Glean from your startup code
+(such as in your application's `application` method or a function called from that method).
```

Comment:
It probably wouldn't hurt to mention _which_ `application` method we recommend, as there are several.
```suggestion
(such as in your application's `UIApplicationDelegate` [`application(_:didFinishLaunchingWithOptions:)`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622921-application) method or a function called from that method).
```